projects/step-frontend/src/lib/modules/timeseries/time-series-context-params.ts

```diff
@@ -5,6 +5,9 @@ export interface TimeSeriesContextParams {
   id: string;
   timeRange: TSTimeRange;
   grouping: string[];
-  baseFilters: any;
+  /**
+   * @deprecated
+   */
+  baseFilters?: any;
```

Comment:
Would removing the deprecated places create to many changes to be integrated into this PR? If yes, I would create a ticket to fix it and reference the ticket ID here

Reply:
I can check

Reply:
@neogucky it can have multiple implications and an overall testing is also needed. I created https://exense.atlassian.net/browse/SED-2042